docs/refs.bib

```diff
@@ -981,6 +996,19 @@ @misc{Sikora_2019_Semidefinite
   url={https://sites.google.com/site/jamiesikora/teaching/sdps-in-quantum-theory}
 }
 
+@article{Sinkhorn_1964_ARelationship,
```

Comment:
Convention is to drop articles (i.e. "a", "the", "an", etc.) and only keep the first word. 

```suggestion
@article{Sinkhorn_1964_Relationship,
```

Reply:
Yes, I would! Thank you!